components/lib/hooks/contracts/useManagerManaged.tsx

```diff
@@ -6,9 +6,9 @@ import {
   DollarMintingCalculator__factory,
   UbiquityAutoRedeem__factory,
   UbiquityGovernance__factory,
-  BondingShareV2__factory,
```

Comment:
Here is where I would draw the line. Do not modify the smart contracts; just import it as BondingV2 and name it stake if you want. Even thought, as mentioned, just changing the text on the screen would have sufficed.